src/consumer.rs

```diff
@@ -644,6 +647,12 @@ impl Builder {
         self
     }
 
+    /// Specifies the security config to use.
+    fn with_security_config(mut self, sec: SecurityConfig) -> Builder {
```

Comment:
* Can this be named `with_security`? Just like `Producer::with_security`; it's shorter and nicer :)
* May I ask you to add a _"See `KafkaClient::new_secured`."_ to the rust doc?